.gitlab-ci.yml

```diff
@@ -415,6 +427,31 @@ examples-contract-build:
       done
     - cargo +stable contract build --manifest-path ./examples/upgradeable-contracts/set-code-hash/updated-incrementer/Cargo.toml
 
+examples-nostd-check:
+  stage:                           examples
+  <<:                              *docker-env
+  <<:                              *test-refs
+  variables:
+    RUSTC_BOOTSTRAP:               "1"
+  script:
+    - rustup component add rust-src --toolchain stable
+    - cargo contract -V
+    - for example in examples/*/; do
+        if [ "$example" = "examples/upgradeable-contracts/" ]; then continue; fi;
+        if [ "$example" = "examples/lang-err-integration-tests/" ]; then continue; fi;
+        pushd $example &&
+        cargo +stable check --no-default-features --target bpfel-unknown-none -Zbuild-std="core,alloc" &&
```

Comment:
I'd pull the target out to a variable, maybe something like `NO_STD_TARGET`. Would make it easier to swap in the future